texteditor/events.go

```diff
@@ -729,5 +729,13 @@ func (ed *Editor) contextMenu(m *core.Scene) {
 			OnClick(func(e events.Event) {
 				ed.Clear()
 			})
+		core.NewButton(m).SetText("Editable").SetIcon(icons.Edit).
+			OnClick(func(e events.Event) {
+				ed.SetReadOnly(false)
```

Comment:
So read-only is nonbinding now? Not sure about that... What about making it conditional on `Generated` since that is the use case? And text should be `Allow editing` or similar in any case.

Reply:
not easy to trace the reason it is readonly at this point. and isn't the proposed new ability "Editable" instead of ReadOnly state??

Reply:
You change the `Generated` property directly below, so I think you could check that... `Editable` is not a verb; `Set editable` or `Make editable` works, as well as `Allow editing`; pick one.